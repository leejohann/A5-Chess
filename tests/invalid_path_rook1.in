move 0 7 0 5
quit
