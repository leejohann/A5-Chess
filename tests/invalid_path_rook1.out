RNBQKBNR
PPPPPPPP
 _ _ _ _
_ _ _ _ 
 _ _ _ _
_ _ _ _ 
pppppppp
rnbqkbnr

it is White's turn
that move is invalid; try again
RNBQKBNR
PPPPPPPP
 _ _ _ _
_ _ _ _ 
 _ _ _ _
_ _ _ _ 
pppppppp
rnbqkbnr

it is White's turn

